elektronLogging/consoleLogger.go

```diff
@@ -0,0 +1,47 @@
+package elektronLogging
+
+import (
+	log "github.com/sirupsen/logrus"
+	"os"
+	"strings"
+)
+
+type ConsoleLogger struct {
+	LoggerImpl
+}
+
+func NewConsoleLogger(logType int, prefix string) *ConsoleLogger {
+	cLog := &ConsoleLogger{}
+	cLog.Type = logType
+	cLog.SetLogFile(prefix)
+	return cLog
+}
+func (cLog *ConsoleLogger) Log(logType int, level log.Level, logData log.Fields, message string) {
```

Comment:
Why the pointer receiver? Technically, this function should not lead to change of state.

Reply:
Yeah, thats right. I will change it